test/normalize.js

```diff
@@ -0,0 +1,202 @@
+
+describe('normalize', function(){
+
+  var normalize = require('../lib/normalize');
+  var list = ['Segment', 'KISSmetrics'];
+  var user = require('../lib/user');
+  var assert = require('assert');
+  var opts;
+  var msg;
+
+  beforeEach(function(){
+    msg = { options: opts = {} };
```

Comment:
Seems too fancy when we can just do:

```js
opts = {};
msg = { options: opts };
```

Which would be clearer at a quick glance.